src/main/java/de/bonndan/nivio/assessment/AssessmentFactory.java

```diff
@@ -0,0 +1,22 @@
+package de.bonndan.nivio.assessment;
+
+import de.bonndan.nivio.model.FullyQualifiedIdentifier;
+import de.bonndan.nivio.model.Landscape;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * A factory to create Assessment instances from map or Landscape
+ */
+
+public class AssessmentFactory {
+
+    public static Assessment createAssessment(Map<FullyQualifiedIdentifier, List<StatusValue>> results) {
+        return new Assessment(results);
+    }
+
+    public static Assessment createAssessment(Landscape landscape) {
```

Comment:
We might think about changing the param to `Map<String, KPI> kpis`, so any combination of assessments can be used.